use_case_examples/tfhers_interoperability/AuthenticationWithDecisionTree.ipynb

```diff
@@ -1,10 +1,31 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction:\n",
+    "\n",
+    "Traditionally, Concrete ML (CML) relied exclusively on the [Concrete]() stack for model compilation. Today, we are expending its scope by introducing compilation support for TFHE-rs, enabling a new level of [interoperability]() between homomorphic encryption frameworks.\n",
```

Comment:
```Traditionally, Concrete ML (CML) relied exclusively on the [Concrete]() stack for model compilation. Today, we are expending its scope by introducing compilation support for TFHE-rs, enabling a new level of [interoperability]() between homomorphic encryption frameworks.```

- don't add links yet, we don't have the docs for this yet
- don't use the "before/now" style -> this is a demo, not a changelog. Suggestion: remove this phrase. and rewrite `Interoperability between Concrete and TFHE-rs allows an application to combine Concrete ML with..`